docs/learn.html

```diff
@@ -100,6 +101,7 @@ <h5 class="my-2">
             <p>
               Download to develop, build and run entirely on the cloud,
               <a href="che-installinfo.html ">step-by-step instructions</a>.
+              <a href="gettingstarted-overview.html">Read more</a> about local, remote and browser-based configurations.
```

Comment:
Add comma after remote in this sentence too. so it reads:
"Read more about local, remote, and browser-based configurations. "